packages/IMAPClient-UI.package/ICFolderDialog.class/instance/modifySeenFlag.st

```diff
@@ -1,7 +0,0 @@
-flags
-modifySeenFlag
-
-	selectedEmail modifyFlag: #seen.
-	selectedEmail seen: (selectedEmail retrieveFlags at: #seen).
-	selectedEmail saveInMaildir.
-	self changed: #emailsOfSelectedFolder
```

Comment:
small refactoring that is not really part of this issue, but it is nice to have:)